features/support/env.el

```diff
@@ -18,6 +18,31 @@
 (require 'ert)
 (require 's)
 
+;;;; Compatibility - From Projectile
+(eval-and-compile
+  ;; Added in Emacs 24.3.
+  (unless (fboundp 'defvar-local)
+    (defmacro defvar-local (var val &optional docstring)
+      "Define VAR as a buffer-local variable with default value VAL.
+Like `defvar' but additionally marks the variable as being automatically
+buffer-local wherever it is set."
+      (declare (debug defvar) (doc-string 3))
+      `(progn
+         (defvar ,var ,val ,docstring)
+         (make-variable-buffer-local ',var))))
+
+  ;; Added in Emacs 24.4
+  (unless (fboundp 'string-suffix-p)
```

Comment:
Well you would have to use the variants found in `s`, the names aren't identical  :)